.gitleaks.toml

```diff
@@ -0,0 +1,8 @@
+[allowlist]
+  description = "Global Allowlist"
+
+  # Ignore based on any subset of the file path
+  paths = [
+    # Ignore all example certs
+    '''\/testing-only-cert-key\/*\.pem$''',
```

Comment:
@lance this will prevent warnings from rh that we've leaked sensitive data due to the testing-only cert/keys added for the kind test setup